src/flight_mode/src/gpio.rs

```diff
@@ -1,8 +1,149 @@
-pub fn init() {
-    println!("flight mode gpio init");
+
+//! GPIO access under Linux using the rust-embedded sysfs_gpio library 
+//! (https://github.com/rust-embedded/rust-sysfs-gpio)
+//!
+//! The methods exposed by this module are centered around
+//! the `MyPins` struct.
+
+use sysfs_gpio::{Pin, Direction};
+use std::collections::LinkedList;
+
+struct Node {
```

Comment:
Jamie: Potentially remove the need to keep num/dir ourselves, use the Pin objects methods of accessing that data. Possibly remove Node altogether